test/test_hip.py

```diff
@@ -23,10 +23,10 @@ def test_has_methods(self):
 
   def test_compile_fail(self):
     with self.assertRaises(RuntimeError):
-      cuda_compile("void test() { {", ["--offload-arch=gfx1100"], HIPCompile, check)
+      compile("void test() { {", ["--offload-arch=gfx1100"], HIPCompile, check)
 
   def test_compile(self):
-    prg = cuda_compile("int test() { return 42; }", ["--offload-arch=gfx1100"], HIPCompile, check)
+    prg = compile("int test() { return 42; }", ["--offload-arch=gfx1100"], HIPCompile, check, filename=None)
```

Comment:
Windows isn't okay with `<null>`?

Reply:
it would throw an error about the profiler not being initialized, not sure how the filename relates to it tho but can debug when I have some spare time